src/web/app/components/question-types/question-edit-details-form/rank-options-question-edit-details-form.component.scss

```diff
@@ -0,0 +1,11 @@
+.cdk-drop-list-dragging .cdk-drag {
```

Comment:
Is there some parent css file that we can move these stylings into since they are duplicated in placed?